test/unit/conftest.py

```diff
@@ -57,6 +61,22 @@ def pytest_ignore_collect(path, config):
     return False
 
 
+def pytest_runtest_setup(item):
+    for mark in item.iter_markers(name='apiver'):
```

Comment:
this is actually really, really, really cool. Maybe write a usage explanation in the docsting? One can derive the meaning of arguments from looking at examples and exceptions in the function, but I think this is something that should be used widely and therefore should have a HOWTO close by.